docs/walkthroughs/setup-mobile-on-existing-cluster.adoc

```diff
@@ -0,0 +1,85 @@
+== Setup Mobile on your existing OpenShift cluster
+
+This document will outline the prerequisites and steps required to try out
+the OpenShift mobile services and mobile client integrations on an existing cluster.
+
+
+=== Pre Requisites 
+
+- Your cluster should be running OpenShift 3.9 or greater
+- The service catalog should be installed
+- The Automation Broker should be installed
+- You should have cluster admin access to your cluster
+- some familiarity with OpenShift
+
+
+
+
+
+=== Set Up
+
+Assuming you have met the prerequisites you can take the following steps
+to install and set up the mobile peices in your OpenShift installation.
+
+All commands were done using the oc tool v3.9.0
+
+
+==== Install the mobile client CRD
+
+Login as the cluster admin. Run the following oc commands.
+
+```
+oc create -f https://raw.githubusercontent.com/aerogear/mobile-cli/master/artifacts/mobileclient_crd.yaml
+oc create clusterrole mobileclient-admin --verb=create,delete,get,list,patch,update,watch --resource=mobileclients
+oc adm policy add-cluster-role-to-group mobileclient-admin system:authenticated
+```
+
+
+==== Configure the Automation Broker
+
+We need to change the broker's config to point to the aerogearcatalog org, we need to set the tag and we also need to change the default sandbox role
+
+The easiest way to do this is via the oc edit command.
+
+You need to know the namespace the broker is installed to. Normally it will be either:
+
+- ansible-service-broker
+- automation-broker
+
+```
+oc edit configmap broker-config -n <broker-ns>
+
+```
+
+Change the ```org``` value under the dockerhub registry key to be ```aerogearcatalog```
+
+Change the ```tag``` value under the dockerhub registry to be the release you want to try (IE 1.0.0)
+
+Change the ```sandbox_role``` value under the openshift section to be ```admin```
+
```

Comment:
@maleck13 not sure about getting the user to modify the default values here since they will lose the services provided by the brokers repo. Might be a better option to tell them to add an additional entry, wdyt?